@types/pal/audio.d.ts

```diff
@@ -0,0 +1,121 @@
+declare module 'pal/audio' {
+    export interface OneShotAudio {
+        /**
+         * stop playing one shot
+         */
+        stop ();
+        /**
+         * register the play finish callback
+         * @param cb
+         */
+        onPlay (cb): OneShotAudio;
+        /**
+         * register the end callback
+         * @param cb
+         */
+        onEnded (cb): OneShotAudio;
+    }
+    export class AudioPlayer {
+        private constructor (nativeAudio: unknown);
+        /**
+         * destroy AudioPlayer
+         */
+        destroy ();
+        /**
+         * load AudioPlayer
+         * @param url
+         * @param opts
+         */
+        static load (url: string, opts?: import('pal/audio/type').AudioLoadOptions): Promise<AudioPlayer>;
+        /**
+         * load native audio for playing one shot
+         * @param url
+         * @param opts
+         */
+        static loadNative (url: string, opts?: import('pal/audio/type').AudioLoadOptions): Promise<any>;
+        /**
+         * max audio channel, if the amount of playing audios exceeds maxAudioChannel, some audio instances should be discarded by audio manager
+         */
+        static maxAudioChannel: number;
```

Comment:
Could this be set by caller? If not, `static readonly maxAudioChannel: number;` is appreciated.

Reply:
yes, it's readonly